packages/react/src/executors/module-federation-dev-server/module-federation-dev-server.impl.ts

```diff
@@ -40,7 +40,8 @@ export default async function* moduleFederationDevServer(
   options: ModuleFederationDevServerOptions,
   context: ExecutorContext
 ): AsyncIterableIterator<{ success: boolean; baseUrl?: string }> {
-  const nxBin = require.resolve('nx');
+  // Force Node to resolve to look for the nx binary that is inside node_modules
+  const nxBin = require.resolve('nx/');
```

Comment:
Use the actual bin path: `nx/bin/nx`

And also, there's also an Angular builder which should be fixed as well.